frontend/src/components/SignIn.js

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+import { useHistory } from 'react-router-dom'
+import {user} from '../reducer/user'
+
+export const SignIn = () => {
+  const [email, setEmail] = useState("")
+  const [password, setPassword] = useState("")
+  const dispatch = useDispatch()
+  const history = useHistory()
+  
+  const handleLogIn = event => {
+    event.preventDefault()
+
+    fetch("https://camilla-and-viktorias-secret.herokuapp.com/sessions",
+      {
+        method: 'POST',
+        headers:{ "Content-Type": "application/json" },
+        body: JSON.stringify({email, password})  
+      })
+      .then (res => {
+        if (!res.ok) {
+          console.log('respons not Ok in signin.js')
+          throw 'Could not log in, try a different username or password'
```

Comment:
If you write like this here:
`throw new Error('Could not log in, try a different username or password')`
you could continue on line 37...